views/metabox/cpcss/container.php

```diff
@@ -28,3 +28,12 @@
 		<?php endif; ?>
 	</div>
 </div>
+<?php if ( $data['async_css_mobile'] ) : ?>
+	<div id="cpcss_response_notice_mobile" class="components-notice is-notice is-warning hidden">
+		<div class="components-notice__content">
+		</div>
+	</div>
+<?php endif; ?>
+<script type="text/javascript">
+	let wprMobileCpcssEnabled = <?php echo esc_js( $data['async_css_mobile'] ); ?>;
+</script>
```

Comment:
This should be added in the `wp_localize_script()` in `AdminSubscriber::admin_enqueue_script()` instead

Reply:
Thank you! I was not aware about this till now. Modified 💯